paper.tex

```diff
@@ -514,6 +514,8 @@ \subsection{List}
 
 The attribute \deff{reversed} is a new array with elements positioned in a reverse order.
 
+The attribute \deff{concat} is a new array combined current \ff{array} and passed array.
```

Comment:
@Graur I would say "a new array that concatenates the current array with the array provided as an argument".